packages/panzer/disc-fe/src/lof/Panzer_EpetraLinearObjFactory_impl.hpp

```diff
@@ -390,17 +390,20 @@ applyDirichletBCs(const LinearObjContainer & counter,
   }
 }
 
-template <typename Traits,typename LocalOrdinalT>
+///////////////////////////////////////////////////////////////////////////////
+//
+//  buildDomainContainer()
+//
+///////////////////////////////////////////////////////////////////////////////
+template <typename Traits, typename LocalOrdinalT>
 Teuchos::RCP<ReadOnlyVector_GlobalEvaluationData>
-EpetraLinearObjFactory<Traits,LocalOrdinalT>::
-buildDomainContainer() const
+EpetraLinearObjFactory<Traits, LocalOrdinalT>::buildDomainContainer() const
 {
-  Teuchos::RCP<EpetraVector_ReadOnly_GlobalEvaluationData> vec_ged
-    = Teuchos::rcp(new EpetraVector_ReadOnly_GlobalEvaluationData);
-  vec_ged->initialize(getGhostedImport(),getGhostedColMap(),getColMap());
-
+  Teuchos::RCP<EpetraVector_ReadOnly_GlobalEvaluationData> vec_ged =
+    Teuchos::rcp(new EpetraVector_ReadOnly_GlobalEvaluationData);
+  vec_ged->initialize(getGhostedImport2(), getGhostedColMap2(), getColMap());
   return vec_ged;
-}
+} // end of buildDomainContainer()
```

Comment:
`buildDomainContainer` now calls version 2 of `getGhostedImport` and `getGhostedColMap`.